### PR TITLE
Rate user with login

### LINKED ISSUE #2126: Rate Skill without login should lead to login page
**Describe the feature you'd like**
Currently, only feedback on skills without logging in takes to login page but rate skill tells that we need to login before rating the skill but does not lead to login page on clicking.

**Additional context**
An Intent so that the rate the skill will lead to login page as well. Because some users might just want to rate but not post any feedback comments.
**Screenshots**

Screenshot_20190401-204227_SUSIAI

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #2126 

Changes: Made changes in SkillDetailsFragment in such a way that the user has to login in case he has to rate a skill.

### CODE REVIEW

**Comment by atm1504** [comment]
@Bhavnaharitsa can you please provide screenshot of your changes. I think there is a bug in this code. Moreover, the app is failing to build in my system. Can you please check that. Thank you : )

**Comment by Bhavnaharitsa** [comment]
> @Bhavnaharitsa can you please provide screenshot of your changes. I think there is a bug in this code. Moreover, the app is failing to build in my system. Can you please check that. Thank you : )

Sure! Are you able to sync fork with master ?

**Comment by atm1504** [comment]
I downloaded the zip version of your branch and tried installing it in my phone. But the build failed.

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt [hunk 1]
```diff
@@ -104,6 +104,7 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
                                 thumbsUp?.setImageResource(R.drawable.thumbs_up_solid)
                                 model.skillLocation?.let { location -> rateSusiSkill(Constant.POSITIVE, location, context) }
                                 setRating(true, true)
+                                Toast.makeText(context,"Thank you for responding", Toast.LENGTH_SHORT).show()
```
Comment (arundhati24) [inline app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]: Please do not use hardcoded strings.

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt [hunk 2]
```diff
@@ -113,6 +114,7 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
                                 thumbsDown?.setImageResource(R.drawable.thumbs_down_solid)
                                 model.skillLocation?.let { location -> rateSusiSkill(Constant.NEGATIVE, location, context) }
                                 setRating(true, false)
+                                Toast.makeText(context,"Thank you for responding", Toast.LENGTH_SHORT).show()
```
Comment (arundhati24) [inline app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]: same.

File: app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt [hunk 3]
```diff
@@ -340,217 +341,223 @@ class SkillDetailsFragment : Fragment(), ISkillDetailsView {
...
+            //Set the bar width
+            //Note : To increase the spacing between the bars set the value of barWidth to < 1f
+            data.barWidth = 0.9f
+            skillRatingChart.setDrawBarShadow(true)
+
+            //Finally set the data and refresh the graph
+            skillRatingChart.data = data
+            skillRatingChart.setViewPortOffsets(0f, 28f, 128f, 28f)
+            skillRatingChart.invalidate()
+        }
```
Comment (arundhati24) [inline app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt]: unnecessary changes.
